readme.md

```diff
@@ -24,12 +24,7 @@ tasklist().then(data => {
 		imageName: 'taskhostex.exe',
 		pid: 1820,
 		sessionName: 'Console',
-		sessionNumber: 1,
-		memUsage: 4415488,  // bytes
-		status: 'Running',
-		username: 'SINDRESORHU3930\\sindre'
-		cpuTime: 0,  // seconds
-		windowTitle: 'Task Host Window'
```

Comment:
I think it's useful for people to see the properties like this with values, so maybe add another example in the `verbose` option section and show these.